packages/execute-driver-plugin/lib/execute-child.js

```diff
@@ -122,11 +135,22 @@ function coerceScriptResult(obj) {
   return obj;
 }
 
-async function main(driverOpts, script, timeoutMs) {
+/**
+ * Entry point to runScript
+ * @param {DriverScriptMessageEvent} eventParams
+ */
+async function main(eventParams) {
```

Comment:
is this function only called internally, so we could easily change its arguments?

Reply:
From what I see, it's only called by `process.on('message', main);` at line 164 and should only trigger by `scriptProc.on('message', res);` in `plugin.js`